crdb/tx.go

```diff
@@ -21,6 +21,7 @@ package crdb
 import (
 	"database/sql"
 
+	"github.com/jinzhu/gorm"
```

Comment:
`gorm` is relatively new, but seems to be the current hotness. I would be hesitant to lock in support for it. A more generic interface would be appreciated. I think that could be achieved by making `executeTxInternal` a public interface and then require users of ORMs such as `gorm` to provide an implementation of `transaction` (as you've done in this PR).